command/agent/cache/handler.go

```diff
@@ -20,7 +20,61 @@ import (
 	"github.com/hashicorp/vault/sdk/logical"
 )
 
-func Handler(ctx context.Context, logger hclog.Logger, proxier Proxier, inmemSink sink.Sink, proxyVaultToken bool) http.Handler {
+func CachelessHandler(ctx context.Context, logger hclog.Logger, proxier Proxier) http.Handler {
```

Comment:
This is another firmly 'take it or leave it' comment:

I'm wondering if the two handlers could have naming that matched up slightly better? I'm not 100% sure I know what that is (`CachingHandler` and `NonCachingHandler` maybe)?

Reply:
Or `CachingProxyHandler` vs `ProxyHandler`?

Reply:
Oh that's a good suggestion! Thanks, I'll go with `CachingProxyHandle` and `ProxyHandler`